modules/ROOT/pages/op-installing-tkn-on-windows.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-windows"]
+
+= Installing {op-title} on Windows
+
+== Binary installation
+
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-windows-amd64.exe[`tkn.exe`] file.
+. Add the location of your `tkn.exe` to the `PATH` variable.
+
```

Comment:
`tkn.exe` file
Here PATH is in back ticks, suggest we use the same style for the Linux and macOS modules too and wherever it is referred to, like on L17

Reply:
done